packages/contentful/migrations/gp2/projects/20230531104312-remove-keywords.js

```diff
@@ -0,0 +1,69 @@
+module.exports.description = 'remove keyword validation';
+
+module.exports.up = (migration) => {
+  const projects = migration.editContentType('projects');
+  projects.editField('keywords').items({
+    type: 'Symbol',
+    validations: [
+      {
```

Comment:
I think we need to keep this. Sets the maximum number you can select.

Reply:
No, this is the validation for each keyword, rather than the validation for the max number of keywords allowed.
See this is within items, not in the keywords array itself.